### PR TITLE
fix: wrong description in Calendar Invite after rescheduling meeting

### LINKED ISSUE #8062: [CAL-1396] Wrong Description after reschedule in Calendar Invite
Screenshot 2023-04-03 at 6 04 50 PM
After reschedule, the calendar invite has `Additional Notes` value as `Description`

<sub>[CAL-1396](https://linear.app/calcom/issue/CAL-1396/wrong-description-after-reschedule-in-calendar-invite)</sub>

### PR DESCRIPTION
**Pull request by sohamdata**
## What does this PR do?

Fixes #8062 

## Type of change

- Bug fix

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,6 +121,9 @@ Here is what you need to be able to run Cal.
    ```sh
    git clone https://github.com/calcom/cal.com.git
    ```
+   > If you are on windows,
+   ```git clone -c core.symlinks=true https://github.com/calcom/cal.com.git``` <br>
+   See [docs](https://cal.com/docs/how-to-guides/how-to-troubleshoot-symbolic-link-issues-on-windows#enable-symbolic-links) for more details.
```
Comment (hariombalhara) [inline README.md:126]:
```suggestion
```

File: packages/features/bookings/lib/handleNewBooking.ts [hunk 2]
```diff
@@ -1553,7 +1553,7 @@ async function handler(
   async function createBooking() {
     if (originalRescheduledBooking) {
       evt.title = originalRescheduledBooking?.title || evt.title;
-      evt.description = originalRescheduledBooking?.description || evt.additionalNotes;
+      evt.description = originalRescheduledBooking?.description || evt.description;
```
Comment (hariombalhara) [inline packages/features/bookings/lib/handleNewBooking.ts:1556]: To me, it seems like the right change.

But this is a change in the core flow and needs confirmation from @zomars @emrysal. Blocking merge till then.

**Review by hariombalhara — DISMISSED** [review]
Left a comment

File: packages/features/bookings/lib/handleNewBooking.ts (hunk 2 above)
Comment (sohamdata) [reply packages/features/bookings/lib/handleNewBooking.ts:1556]: any update?

**Review by alannnc — APPROVED** [review]

File: packages/features/bookings/lib/handleNewBooking.ts (hunk 2 above)
Comment (PeerRich) [reply packages/features/bookings/lib/handleNewBooking.ts:1599]: @hariombalhara @alannnc is this safe to merge?

Comment (alannnc) [reply packages/features/bookings/lib/handleNewBooking.ts:1599]: I think back then was some kind of error and description was arriving on additionalNotes? Something like that... I believe this is the right thing to do now so Safe to merge. @PeerRich

**Pull request merged**